src/Http/Http/src/DefaultHttpContext.cs

```diff
@@ -40,13 +46,21 @@ public DefaultHttpContext()
             Features.Set<IHttpResponseBodyFeature>(new StreamResponseBodyFeature(Stream.Null));
         }
 
+        /// <summary>
+        /// Initializes a new instance of the DefaultHttpContext class with options passed in.
+        /// <summary/>
+        /// <param name="features">Options to set when instantianting the Default HTTP context object.</param>
         public DefaultHttpContext(IFeatureCollection features)
         {
             _features.Initalize(features);
             _request = new DefaultHttpRequest(this);
             _response = new DefaultHttpResponse(this);
         }
 
+        /// <summary>
+        /// Initialize the current instant of the class with options passed in.
+        /// <summary/>
```

Comment:
Let's add some remarks in. This one's a little tricky.

```
        /// <remarks>
        /// This method allows the consumer to re-use the <see cref="DefaultHttpContext" /> for another request, rather than having to allocate a new instance.
        /// <remarks>
```